packages/dx-react-scheduler/src/plugins/appointment-form.tsx

```diff
@@ -234,7 +260,7 @@ export class AppointmentForm extends React.PureComponent {
         </Template>
 
         <Template name="appointment">
-          {params => (
+          {(params: any) => (
```

Comment:
params type is Appointments.AppointmentProps
... same for other Templates with parameters

Reply:
Sometimes, components props are not equal with template params

Reply:
But they should be strongly typed when equal